python/lsst/obs/base/instrument_tests.py

```diff
@@ -27,11 +27,92 @@
 
 import abc
 import dataclasses
+import pkg_resources
 
-from lsst.obs.base import Instrument
+from lsst.obs.base import Instrument, FilterDefinitionCollection, FilterDefinition
 from lsst.obs.base.gen2to3 import TranslatorFactory
+from lsst.obs.base.yamlCamera import makeCamera
 from lsst.daf.butler import Registry
 from lsst.daf.butler import RegistryConfig
+from lsst.daf.butler.core.utils import getFullTypeName
+from lsst.daf.butler.formatters.yaml import YamlFormatter
+
+from .utils import createInitialSkyWcsFromBoresight
+
+DUMMY_FILTER_DEFINITIONS = FilterDefinitionCollection(
+    FilterDefinition(physical_filter="dummy_u", band="u", lambdaEff=0),
+    FilterDefinition(physical_filter="dummy_g", band="g", lambdaEff=0),
+)
+
+
+class TestYamlFormatter(YamlFormatter):
+    """Specialist formatter for tests that can make a WCS"""
+
+    @classmethod
+    def makeRawSkyWcsFromBoresight(cls, boresight, orientation, detector):
+        """Class method to make a raw sky WCS from boresight and detector.
+
+        Parameters
+        ----------
+        boresight : `lsst.geom.SpherePoint`
+            The ICRS boresight RA/Dec
+        orientation : `lsst.geom.Angle`
+            The rotation angle of the focal plane on the sky.
+        detector : `lsst.afw.cameraGeom.Detector`, unused
+            Where to get the camera geomtry from.
+
+        Returns
+        -------
+        skyWcs : `~lsst.afw.geom.SkyWcs`
+            Reversible mapping from pixel coordinates to sky coordinates.
+
+        Notes
+        -----
+        This makes no attempt to create a proper WCS from geometry.
```

Comment:
Is most of this docstring identical to the parent except for the Notes? Since this is just here as a test helper and isn't in `__all__`, I wonder if making the docstring short and explicit about that, while referencing the parent class is better?